apis/nucleus/src/components/listbox/components/ListBoxRowColumn.jsx

```diff
@@ -258,10 +266,27 @@ function RowColumn({ index, rowIndex, columnIndex, style, data }) {
     };
   } else {
     cellIndex = index;
+    count.max = listCount;
+    count.currentIndex = index;
     styles = { ...style };
   }
+  cellIndex += dataOffset;
 
-  const handleKeyDownCallback = useCallback(getFieldKeyboardNavigation(actions), [actions]);
+  const [rowRef, setRowRef] = useState(null);
```

Comment:
Is there a reason for not using a `useRef` here?

Reply:
Changed to `useRef` 👍